### DISCUSSION TITLE
mqtt subscribe code examples

### DISCUSSION THREAD

**Discussion by livezen** [arendst/Tasmota#20396] [Q&A]
hi,

can maybe somebody hint some example code for mqtt subscription inside tasmota code. 
 if (Mqtt.connected)
        MqttSubscribe("tasmota/myproject");
 
how to handle the response from XdrvMailbox.data?

thanks!

**Comment by barbudor** [comment]
You are very vague
drivers are not supposed to make their own MQTT subscriptions as there is already an structure in Tasmota to manage incoming topics
You may look how Berry is plugged on this
